netutils/config/compliance.py

```diff
@@ -172,14 +180,16 @@ def compliance(features, backup, intended, network_os, cfg_type="file"):
     compliance_results = {}
 
     for feature in features:
-        backup_str = section_config(feature, backup_cfg, network_os)
-        intended_str = section_config(feature, intended_cfg, network_os)
+        backup_str = section_config(feature, backup_cfg, network_os)  # type: ignore
+        intended_str = section_config(feature, intended_cfg, network_os)  # type: ignore
```

Comment:
This could be fixed by changing the return type of `open_file_config` to just a `str`.